src/components/layout/index.js

```diff
@@ -52,9 +51,7 @@ const Layout = ({
       />
       <main id="main">
         <SkipNavContent />
-        <Container narrow={narrow} centered={centered}>
-          {children}
-        </Container>
+        <Container centered={centered}>{children}</Container>
```

Comment:
Can we just use `centered` here instead of `centered={centered}`?

Reply:
Nope! It's passing a prop down from `<Layout centered>` down to `Container`